cupyx/fallback_mode/ndarray.py

```diff
@@ -0,0 +1,130 @@
+"""
+class ndarray is wrapper around cupy.ndarray
+to support fallback of methods of type `ndarray.func()`
+"""
+import numpy as np
+
+import cupy as cp
+from cupyx.fallback_mode import data_transfer
+
+try:
+    from cupyx.fallback_mode import fallback
+except ImportError:
```

Comment:
Avoid cyclic imports without suppressing `ImportError.

Reply:
Are imports inside functions permitted?

Reply:
No. Please change the location of function declarations.

Reply:
`fallback.py`, `utils.py`, `ndarray.py` these are all inter-dependent on one another. I think we need to combine this into single file then.

Reply:
It seems OK to combine.